src/main/java/com/enterprise/eventmanagementsystem/dao/EventSQLDAO.java

```diff
@@ -20,7 +17,7 @@ public Event fetch(int eventId) {
     }
 
     @Override
-    public Iterable<Event> fetchAll() throws Exception {
+    public Iterable<Event> fetchAll() {
         return eventRepository.findAll();
     }
 
```

Comment:
throws Exception was being unused for now there isn't a need for it to be implemented until there is actual exception handling that is being used.

Reply:
Good.

Since Exception is the superclass of runtime exceptions, it's a given that all methods have throws Exception.  Thus, declaring it explicitly doesn't add value.